src/test/ToolTip.test.js

```diff
@@ -0,0 +1,125 @@
+    mytooltip.init = jest.fn(mytooltip.init);
+    mytooltip.init(tooltipInfoArr);
+
+    expect(mytooltip.init).toHaveBeenCalledWith(tooltipInfoArr);
+  });
+
+  test('Should edit tooltips when call "edit" method', () => {
+    mytooltip.init(tooltipInfoArr);
+    mytooltip.edit = jest.fn(mytooltip.edit);
+    mytooltip.edit('#first', {
+      contents: 'labore et dolore magna aliqua',
+      delay: 100
+    });
+
+    expect(mytooltip.edit).toHaveBeenCalledWith('#first', {
+      contents: 'labore et dolore magna aliqua',
+      delay: 100
+    });
+  });
+
+  test('Should modify tooltip HTMLElement edited textContent', () => {
+    mytooltip.init(tooltipInfoArr);
+    mytooltip.edit = jest.fn(mytooltip.edit);
+
+    const firstTooltip = tooltipInfoArr[0];
+    const createdTooltip = mytooltip.edit(firstTooltip.element, {
+      contents: 'edited contents',
+      delay: 100
+    });
+
+    expect('edited contents').toBe(createdTooltip.textContent);
+  });
+
+  test('Should throw error "element was not exist element" when call edit method with not exist element', () => {
+    mytooltip.init(tooltipInfoArr);
+    mytooltip.edit = jest.fn(mytooltip.edit);
+
+    expect(() => mytooltip.edit('#han', {
+      contents: 'edited contents',
+      delay: 4000
+    })).toThrow(`#han was not exist element in tooltipMap`);
+  });
+
+  test('Should add tooltip when call "add" method', () => {
+    mytooltip.add = jest.fn(mytooltip.add);
+
+    const addTooltipInfo = {
+      contents: 'test image',
+      delay: 300
+    };
+    const createdTooltip = mytooltip.add('.my-img', addTooltipInfo);
+
+    expect(addTooltipInfo.contents).toBe(createdTooltip.textContent);
+    expect(createdTooltip.getAttribute('class')).toEqual(expect.stringContaining('tooltip-content'));
+  });
+
+  test('Should throw error "element was already exist element" when call add method with already exist element', () => {
+    mytooltip.init(tooltipInfoArr);
+    mytooltip.add = jest.fn(mytooltip.add);
+
+    const firstTooltipInfo = tooltipInfoArr[0];
+
+    expect(() => mytooltip.add(firstTooltipInfo.element, {
+      contents: firstTooltipInfo.contents,
+      delay: firstTooltipInfo.delay
+    })).toThrow(`${firstTooltipInfo.element} was already exist element in tooltipMap`);
+  });
+
+  test('Should remove tooltip when call "remove" method', () => {
+    mytooltip.init(tooltipInfoArr);
+    mytooltip.remove = jest.fn(mytooltip.remove);
+
+    const firstTooltipInfo = tooltipInfoArr[0];
+    const deletedElementTooltip = mytooltip.remove(firstTooltipInfo.element);
+
+    expect(firstTooltipInfo.element).toBe(deletedElementTooltip);
+  });
+
+  test('Should throw error "element was not exist element" when call remove method with not exist element', () => {
+    mytooltip.init(tooltipInfoArr);
+
+    mytooltip.remove = jest.fn(mytooltip.remove);
+
+    expect(() => mytooltip.remove('#han').toThrow(`#han was not exist element in tooltipMap`));
```

Comment:
expect() 이후 실행되는 결과가 없어요! 이 테스트가 잘 통과하나요? 👀

```suggestion
    expect(() => mytooltip.remove('#han')).toThrow(`#han was not exist element in tooltipMap`);
```